components/properties/src/provider.rs

```diff
@@ -47,7 +47,7 @@ pub enum UnicodePropertyV1<'data> {
 )]
 #[cfg_attr(feature = "serde", derive(serde::Deserialize))]
 #[non_exhaustive]
-pub enum UnicodePropertyMapV1<'data, T: TrieValue> {
+pub enum PropertyUnicodeMapV1<'data, T: TrieValue> {
```

Comment:
Issue: This is also only a `CodePointMap`, because CodePointTrie maps from code points to values. If we wanted to support strings, `UnicodeMap`, it would likely be backed by a Char16Trie, not a CodePointTrie.

Reply:
Got it! Switched `UnicodePropertyMapV1 -> PropertyCodePointMapV1`